src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_PatternSwitchStatement.cs

```diff
@@ -49,7 +49,15 @@ public override BoundNode VisitPatternSwitchStatement(BoundPatternSwitchStatemen
             }
 
             result.Add(_factory.Label(node.BreakLabel));
-            var translatedSwitch = _factory.Block(pslr.DeclaredTemps.ToImmutableArray().Concat(node.InnerLocals), node.InnerLocalFunctions, result.ToImmutableAndFree());
+            BoundStatement translatedSwitch = _factory.Block(pslr.DeclaredTemps.ToImmutableArray().Concat(node.InnerLocals), node.InnerLocalFunctions, result.ToImmutableAndFree());
```

Comment:
I would prefer to leave that issue to someone who understands what's going on better than I.